src/Mvc/Extensions.ApiDescription.Client/src/Microsoft.Extensions.ApiDescription.Client.csproj

```diff
@@ -7,7 +7,7 @@
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <NuspecFile>$(MSBuildProjectName).nuspec</NuspecFile>
     <PackageId>$(MSBuildProjectName)</PackageId>
-    <PackageTags>Build Tasks;MSBuild;Swagger;Open API;code generation; Web API client</PackageTags>
+    <PackageTags>Build Tasks;MSBuild;Swagger;OpenAPI;code generation; Web API client</PackageTags>
```

Comment:
Ha, didn't realize you changed these files. Please revert before merging because I have a PR out that does this _and_ adds more tags to the lines